core/snippets/snippets.py

```diff
@@ -158,6 +175,10 @@ def create_lists(
 
 def on_ready():
     fs.watch(str(SNIPPETS_DIR), lambda _1, _2: update_snippets())
+
+    if get_setting_dir():
```

Comment:
since this uses a setting, wouldn't the user have to restart talon after changing it as-is? I'm not sure if there's a way around this at the moment.  

It's also a little unique in that I don't _think_ it makes sense to change the setting per context, so we may wish to add a note to the readme/code.

Reply:
Correct

I added a comment to the settings file